diff --git a/node_modules/rtcpeerconnection/rtcpeerconnection.js b/node_modules/rtcpeerconnection/rtcpeerconnection.js
index ccc7d94..c5ac276 100644
--- a/node_modules/rtcpeerconnection/rtcpeerconnection.js
+++ b/node_modules/rtcpeerconnection/rtcpeerconnection.js
@@ -783,7 +783,7 @@ PeerConnection.prototype._answer = function (constraints, cb) {
 // Internal method for emitting ice candidates on our peer object
 PeerConnection.prototype._onIce = function (event) {
     var self = this;
-    if (event.candidate) {
+    if (event.candidate && event.candidate.candidate.length) {
         if (this.dontSignalCandidates) return;
         var ice = event.candidate;
 
